cpp/src/io/json/json_tree.cu

```diff
@@ -736,13 +825,17 @@ rmm::device_uvector<size_type> compute_row_offsets(rmm::device_uvector<NodeIndex
 std::tuple<rmm::device_uvector<NodeIndexT>, rmm::device_uvector<size_type>>
 records_orient_tree_traversal(device_span<SymbolT const> d_input,
                               tree_meta_t const& d_tree,
+                              bool is_array_of_arrays,
+                              bool is_enabled_lines,
                               rmm::cuda_stream_view stream,
                               rmm::mr::device_memory_resource* mr)
 {
   CUDF_FUNC_RANGE();
-  auto [new_col_id, new_parent_col_id] = generate_column_id(d_input, d_tree, stream, mr);
+  auto [new_col_id, new_parent_col_id] =
```

Comment:
```suggestion
  auto [new_col_ids, new_parent_col_ids] =
```

Reply:
`new_col_id` is moved, it can't be const.